### PR TITLE
LPS-119208

### PR DESCRIPTION
**Pull request by jbalsas**
This is an initial **DRAFT PULL REQUEST** to showcase the final changes requested by UX for [UX Refinement for Applications Menu](https://issues.liferay.com/browse/LPS-117782)

This PR includes:
- Removal of the Company Logo Control Menu Entry (the company logo no longer shows inside the control menu for control panel apps)
- Duplication of the ApplicationsMenu Control Menu Entry so it shows in the Users section when in a regular layout and on the Sites section when inside control panel apps
- Adds the portlet category to the portlet header on top of the portlet title
- Shows a dropdown with all the apps inside the same panel category

Screen Shot 2020-08-18 at 17 47 18

Pending work:
- Visuals will be adjusted by @eduardoallegrini and/or @marcoscv-work
- Backend code needs to be properly organized... @ealonso, could you see that someone in the team does that before we push forward?

### CODE REVIEW

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/portlet_header.jsp [hunk 1]
```diff
@@ -17,9 +17,74 @@
 <%@ include file="/init.jsp" %>
 
 <%
+String portletId = portletDisplay.getRootPortletId();
```
Comment (jbalsas) [inline modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/portlet_header.jsp:20]: We probably want to move this somewhere... I just hacked it here to try to see this working. I can move it somewhere else if you let me know where.

File: modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/product/navigation/control/menu/ApplicationsMenuSiteProductNavigationControlMenuEntry.java [hunk 2]
```diff
@@ -50,7 +51,7 @@
 	},
 	service = ProductNavigationControlMenuEntry.class
 )
-public class ApplicationsMenuProductNavigationControlMenuEntry
+public class ApplicationsMenuSiteProductNavigationControlMenuEntry
```
Comment (jbalsas) [inline modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/java/com/liferay/product/navigation/applications/menu/web/internal/product/navigation/control/menu/ApplicationsMenuSiteProductNavigationControlMenuEntry.java:54]: The diff is weird in this UI, but basically all I did was:
- Duplicate `ApplicationsMenuProductNavigationControlMenuEntry`
- Call them:
    - ApplicationsMenuSiteProductNavigationControlMenuEntry
    - ApplicationsMenuControlPanelProductNavigationControlMenuEntry
- Add the `layout.isTypeControlPanel()` checks to their `isShow` (they have inverted values).

I tried quickly creating a `Base`, but we no longer allow doing so with things that have an `@Reference`, so I don't really know what's the proper pattern for t his :(

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
see: https://github.com/jbalsas/liferay-portal/pull/2230

**Comment by ealonso** [comment]
ci:close

**Pull request closed without merging**